packages/abi-gen/test-cli/fixtures/contracts/AbiGenDummy.sol

```diff
@@ -135,7 +137,28 @@ contract AbiGenDummy
     function nestedStructInput(NestedStruct memory n) public pure {}
     function nestedStructOutput() public pure returns(NestedStruct memory) {}
 
+    struct StructNotDirectlyUsedAnywhere {
+        uint256 aField;
+    }
+
+    struct NestedStructWithInnerStructNotUsedElsewhere {
+        StructNotDirectlyUsedAnywhere innerStruct;
+    }
+
+    function methodUsingNestedStructWithInnerStructNotUsedElsewhere()
+        public pure returns(NestedStructWithInnerStructNotUsedElsewhere  memory)
+    {}
+
     uint someState;
     function nonPureMethod() public returns(uint) { return someState += 1; }
     function nonPureMethodThatReturnsNothing() public { someState += 1; }
+
+    function methodReturningMultipleValues()
```

Comment:
In line with `structOutput` and `structArrayOutput`, maybe this one could be `tupleOutput`?